save-backend/src/test/kotlin/org/cqfn/save/backend/controller/AgentsControllerTest.kt

```diff
@@ -59,4 +59,17 @@ class AgentsControllerTest {
                 Assertions.assertEquals(AgentState.BUSY, statuses[1]!!.state)
             }
     }
+
+    @Test
+    fun `should return error not found container id`() {
```

Comment:
I thought you'd add a test in `JpaSpecificationTest`, just to test that a specific exception is thrown